app/src/main/java/com/boolder/boolder/view/search/TickViewModel.kt

```diff
@@ -0,0 +1,66 @@
+package com.boolder.boolder.view.search
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.boolder.boolder.R
+import com.boolder.boolder.data.database.repository.TickRepository
+import com.boolder.boolder.domain.convert
+import kotlinx.coroutines.launch
+import java.text.Normalizer
+
+class TickViewModel(
+    private val tickRepository: TickRepository,
+) : ViewModel() {
+
+    private val _result = MutableLiveData<List<BaseObject>>()
+    val searchResult: LiveData<List<BaseObject>> = _result
+
+    fun list(){
+        viewModelScope.launch {
+//            tickRepository.deleteAll()
+            val areas_with_problems = tickRepository.getProblemsPerArea();
```

Comment:
Side note: the Kotlin code style states that the value names should be written in camel case 😄

Reply:
Thank you for the feedback! I'll have a go at processing the comments later this week

Reply:
Changed it to camel case :smile: